pkg/workloads/cortex/lib/storage/s3.py

```diff
@@ -249,3 +249,9 @@ def get_json_external(self, s3_path, num_retries=0, retry_delay_sec=2):
         if obj is None:
             return None
         return json.loads(obj.decode("utf-8"))
+
+    def list_objects(self, path):
```

Comment:
I think list_object should take in the path without the leading `s3://<bucket_name>`, since the client already knows the bucket, what do you think?